coralproject/talk
# Collapse replies

## What does this PR do?

- clicking the Reply button on another comment will collapse the previously opened Reply Box.
- adds a Cancel button to the Reply Box.

## How do I test this PR?

- click the Reply Button on a comment
- click a different Reply Button. the prior one should close
- enter some text in the reply field. click around. the text should stay
- post the comment. the reply box should collapse.
